mappings/net/minecraft/class_4719.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_4719
```

Comment:
This is like a sign type, but dunno if mojang will use it for other wood stuff in the future...